### PR TITLE
See number of players in room

### PR DESCRIPTION
**Pull request by bumblebot1**
Improved styling of lobby page to match the ideas discussed in sprint plan;also added number of players in room to each room displayed

### CODE REVIEW

**Comment by JamieZafar** [comment]
https://www.signwarehouse.com/blog/wp-content/uploads/2011/01/ReduceDebt_Fotolia_17236102_XS.jpg

File: src/client/lobby/user-list.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ export default function UserList(props) {
     return (
         <div className="col-lg-4 text-center">
             <h2>User List</h2>
-            <div className={`pre-scrollable ${style['list-areas']}`}>
+            <div className="col-lg-12 pre-scrollable list-group">
                 {props.lobbyUsers.map( result => {
                     return (
-                        <p className='row' key={result['id']}>
+                        <li className={`list-group-item list-group-item-success ${style.space}`} key={result['id']}>
```
Comment (lclarksl) [inline src/client/lobby/user-list.js]: YES

Comment (lclarksl) [inline src/client/lobby/user-list.js]: Although these should be inside a `ul`

File: src/client/lobby/room-list.spec.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ describe('<RoomList />', () => {
 
     it('renders the list of open rooms correctly', () => {
         const wrapper = shallow(<RoomList roomList={mockList} />);
-        wrapper.find('p').forEach((current, index) => {
-            expect(current.text()).toEqual(mockList[index]['name']);
+        wrapper.find('button').forEach((current, index) => {
+            expect(current.text()).toEqual(mockList[index]['name']+mockList[index]['users'].length);
```
Comment (lclarksl) [inline src/client/lobby/room-list.spec.js]: Use ES6 `${}` for string concatenation.
Also better to have `mockList[index].name` and `mockList[index].users`

File: src/client/lobby/user-list.spec.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ describe('<UserList />', () => {
 
     it('renders the list of users in lobby correctly', () => {
         const wrapper = shallow(<UserList lobbyUsers={mockList} />);
-        wrapper.find('p').forEach((current, index) => {
+        wrapper.find('li').forEach((current, index) => {
             expect(current.text()).toEqual(mockList[index]['name']);
```
Comment (lclarksl) [inline src/client/lobby/user-list.spec.js]: `mockList[index].name`

Comment (JamieZafar) [inline src/client/lobby/user-list.spec.js]: should we change this in all components?

Comment (lclarksl) [inline src/client/lobby/user-list.spec.js]: Yeah. Consistency is ood

Comment (lclarksl) [inline src/client/lobby/user-list.spec.js]: Consistency is also good

Comment (JamieZafar) [inline src/client/lobby/user-list.spec.js]: http://www.bbc.co.uk/doctorwho/medialibrary/images/1024/s4_03_wal_08.jpg?size=1024&promo=/doctorwho/medialibrary/images/main-promo/s4_03_wal_08.jpg&purpose=Computer%20wallpaper&summary=Out%20in%20the%20cold.&info=&tag_file_id=s4_03_wal_08

**Pull request merged**